ray-operator/config/samples/pytorch-resnet-image-classifier/ray-job.pytorch-image-classifier.yaml

```diff
@@ -85,7 +85,7 @@ spec:
               effect: "NoSchedule"
             containers:
               - name: ray-worker
-                image: rayproject/ray-ml:2.9.0-gpu
+                image: rayproject/ray-ml:2.41.0.021baf-py311
```

Comment:
In this case, the worker Pod may use a different Python version than the head Pod.

Reply:
Good point, this was the only manifest I wasn't able to test cause my cluster didn't have GCSFuse enabled. Validated this works when I use `rayproject/ray-ml:2.41.0.021baf-py39`